### DISCUSSION TITLE
All items of all loaded bundles visible at once

### DISCUSSION THREAD

**Discussion by SFGrenade** [nesrak1/UABEA#138] [Ideas]
As the title suggests, a setting to display all items of all loaded bundles would be great for data dumping (like the original UABE allows one to do).

**Comment by nesrak1** [comment]
Do you mean the assets file dependency list? Original UABE I don't think could load multiple bundles at once.

**Comment by SFGrenade** [comment]
oh, might've worded poorly.
i meant that in orig UABE i can load several, if not all, level# and sharedassets#.assets files and see all contained assets in a list.

**Comment by nesrak1** [comment]
Oh, did it have multiselect in the open file menu? I can add that pretty easily then.

**Comment by SFGrenade** [comment]
yes, it does have that.
